addons/mail_plugin/__manifest__.py

```diff
@@ -3,7 +3,7 @@
 
 {
     'name': 'Mail Plugin',
-    'version': '1.0',
+    'version': '1.1',
```

Comment:
Is it strictly necessary? It's already a new module (ok, actually renamed) in this version.

Reply:
Not strictly necessary, we can stay on 1.0 if you want

Reply:
I agree we may keep 1.0 :)